scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -201,8 +168,27 @@ task Filter {
         GATK_JAR=${gatk4_jar_override}
     fi
 
-  	java -Xmx4g -jar $GATK_JAR FilterMutectCalls -V ${unfiltered_calls} -O ${output_prepend}-filtered.vcf \
-       ${"-contaminationTable " + contamination_table}
+    if [[ "${variants_for_contamination}" == *.vcf ]]; then
+        java -Xmx4g -jar $GATK_JAR GetPileupSummaries -I ${tumor_bam} -L ${intervals} -V ${variants_for_contamination} -O pileups.table
```

Comment:
I can be convinced that combining these into one WDL task is a good idea .... But here are my concerns:

1)  At some point, we are going to want to allow users to specify a sequencing artifact metric file (optional) so that they do not have to rerun this task.  

2)  How long do these tasks take?  The SequencingArtifactMetrics can take a while.  I worry about failures costing a lot in terms of lost time.  

3)  Some of these can run in parallel, which save time.

What're your thoughts?